fuzzit.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -xe
+
+# go-fuzz doesn't support modules yet, so ensure we do everything
+# in the old style GOPATH way
+export GO111MODULE="off"
+
+# install go-fuzz
+go get -u github.com/dvyukov/go-fuzz/go-fuzz github.com/dvyukov/go-fuzz/go-fuzz-build
+
+# target name can only contain lower-case letters (a-z), digits (0-9) and a dash (-)
+# to add another target, make sure to create it with `fuzzit create target`
+# before using `fuzzit create job`
+TARGETS=("promql-parse-metric" "promql-parse-open-metric" "promql-parse-metric-selector" "promql-parse-expr")
+
+go-fuzz-build -libfuzzer -func FuzzParseMetric -o promql-parse-metric.a ./promql
+go-fuzz-build -libfuzzer -func FuzzParseOpenMetric -o promql-parse-open-metric.a ./promql
+go-fuzz-build -libfuzzer -func FuzzParseMetricSelector -o promql-parse-metric-selector.a ./promql
+go-fuzz-build -libfuzzer -func FuzzParseExpr -o promql-parse-expr.a ./promql
+
+for TARGET in "${TARGETS[@]}"
+do
+    clang -fsanitize=fuzzer ${TARGET}.a -o ${TARGET}
```

Comment:
Could you add more comments here, and also for the `go-fuzz-build` steps above, to explain what their purpose is (for someone who isn't familiar with all the fuzz tools and steps of fuzzing)?

Reply:
done.